package.json

```diff
@@ -10,7 +10,7 @@
     "smokehouse": "./cli/test/smokehouse/frontends/smokehouse-bin.js"
   },
   "engines": {
-    "node": ">=14.15"
+    "node": ">=16.17"
```

Comment:
Not much reasoning behind this. 16.17.1 is latest as of today. I don't know what feature sets are necessary / how low we could go. Picked the latest seems reasonable.

https://nodejs.org/en/download/releases/